contracts/oracles/OraclePoolToken.sol

```diff
@@ -96,7 +96,8 @@ contract OraclePoolToken is IOracleUsd {
             uint a = aPool.mul(eAvg);
             uint b = a.mul(9).div(Q112);
             uint c = ePool.mul(3988000);
-            uint sqRoot = sqrt(a.div(Q112).mul(b.add(c)));
+            //uint sqRoot = sqrt(a.div(Q112).mul(b.add(c)));
+            uint sqRoot = b.add(c).mul(sqrt(a)).div(sqrt(Q112));
```

Comment:
I think there is changed order of operations since the entire value should be square root. I've updated the setup with latest solidity and no SafeMath, we need to update this and check if the tests pass